nav2_msgs/srv/GetCosts.srv

```diff
@@ -1,7 +1,7 @@
 # Get costmap costs at given poses
 
 bool use_footprint
-geometry_msgs/PoseStamped[] poses
+nav2_msgs/PoseStampedArray poses
```

Comment:
Q: Does this one need to be changed? I don't mind it, but I view this more as the `path.poses` list as its usual use-case and I'm not sure I see the need for the header for this particular application